enaml/qt/qt_mpl_canvas.py

```diff
@@ -9,18 +9,26 @@
 
 from enaml.widgets.mpl_canvas import ProxyMPLCanvas
 
-from matplotlib.backends.backend_qt4agg import FigureCanvasQTAgg
-try:
-    from matplotlib.backends.backend_qt4agg import (NavigationToolbar2QTAgg
-                                                    as NavigationToolbar2QT)
-except ImportError:
-    from matplotlib.backends.backend_qt4agg import NavigationToolbar2QT
-
-from .QtCore import Qt
+from qtpy import QT_VERSION
+from .QtCore import Qt, PYQT_VERSION_STR
```

Comment:
PYQT_VERSION_STR is unused so please remove it. Also you can avoid importing QT_VERSION from qtpy by importing __version__ or __version_info__ (tuple of int) from .QtCore which I would prefer.